src/Tribe/Tickets.php

```diff
@@ -683,6 +693,10 @@ public function __construct() {
 
 			// Ensure ticket prices and event costs are linked
 			add_filter( 'tribe_events_event_costs', array( $this, 'get_ticket_prices' ), 10, 2 );
+			// Flush caches if any of the following actions has occurred
+			add_action( 'save_post', array( $this, 'clear_cache' ) );
```

Comment:
I'm _slightly_ concerned that this action will run on save post for all posts no matter what. I'd be tempted to check that the post we are acting on is of a post type that could have tickets. It's probably OK though. If the `delete_transient` function was less complicated I wouldn't have even raised the concern

Reply:
As any post type can have tickets you're right we need to check before attaching the hooks to know on which post types we only need to clear caches, however as every post has a unique ID and this ID is associate independently from each other by clearing or removing a cache on save post even if the post does not have the cache removed is only the one associated with the post being modified. 

Let me know what do you think and if you think is best to do it dynamically based on the options from the admin on the post types that has tickets associated with it.